fixtures/neb.py

```diff
@@ -14,3 +14,11 @@ def neb_env(request):
     neb_env = config.getoption("neb_env") or config.getini("neb_env")
     if neb_env is not None:
         return neb_env
+
+@pytest.fixture(scope="session")
+def insecure(request):
+    """Returns the insecure flag setting"""
+    config = request.config
+    insecure = config.getoption("insecure")
+    if insecure is not None:
+        return insecure
```

Comment:
```suggestion
    return insecure
```
If insecure is `None`, the function will not explicitly return, which implicitly returns `None`. This is the same as just returning the value whether it is `None` or not.

Alternatively,
```suggestion
    if insecure is not None:
        return insecure
    return False
```
might be neater depending on your viewpoint.